src/chargen.cpp

```diff
@@ -1795,5 +1833,6 @@ void create_parse(struct descriptor_data *d, const char *arg)
       }
     start_game(d);
     break;
+    }

```

Comment:
I think this guy's out of place. Should be at the end of your CCR_MAGE ifcheck around line line 1789, right? Does Chargen work as expected for shamans etc?

Reply:
Unsure. I've run through CG for both full shamans and aspected shamans (even checked adepts just to be safe) a few times on my end and had no issues

Reply:
Ran it both ways, compiled and didn't find any issues either way, so best to follow your hunch and go with it